Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM python:3.11.9-slim
+
+ENV GO_VERSION=1.22.1
```

Comment:
```suggestion
ENV GO_VERSION=1.22.2
```

Reply:
Bumped along in [`494673d`](https://github.com/agriyakhetarpal/hugo-python-distributions/pull/75/commits/494673ddf955125707f02a5078939e74ac3c7f78)